chaoskube/chaoskube_test.go

```diff
@@ -148,15 +153,41 @@ func TestCandidatesNamespaces(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		chaoskube := setup(t, labels.Everything(), labels.Everything(), namespaces, false, 0)
+		chaoskube := setup(t, labels.Everything(), labels.Everything(), namespaces, labels.Everything(), false, 0)
+
+		validateCandidates(t, chaoskube, test.pods)
+	}
+}
+
+func TestCandidatesNamespaceLabels(t *testing.T) {
+	foo := map[string]string{"namespace": "default", "name": "foo"}
+	bar := map[string]string{"namespace": "testing", "name": "bar"}
+
+	for _, test := range []struct {
+		namespaceLabel string
+		pods           []map[string]string
+	}{
+		{"", []map[string]string{foo, bar}},
+		{"doesnotexist", []map[string]string{foo, bar}},
```

Comment:
I would expect this to not find any pod as there's no namespace that satifies this label selector.